test/functional/ui/screen_basic_spec.lua

```diff
@@ -73,33 +73,29 @@ describe('Screen', function()
   describe(':suspend', function()
     it('is forwarded to the UI', function()
       local function check()
-        if not screen.suspended then
-          return 'Screen was not suspended'
-        end
+        eq(true, screen.suspended)
       end
```

Comment:
Not for me. It passes, with a warning about "Warning: Screen changes have been received after the expected state was seen."

In `Screen:expect(()` we only check `pcall()`, and test failures don't raise Lua errors. It would be great if we could check test failures, I haven't found way to do that though (may require digging into luassert to see where it tracks test failures)